### ISSUE TITLE
Packaging must specify libraries used

### ISSUE THREAD

**Issue by longshorej** [lightbend/reactive-cli#15]

**Comment by longshorej** [comment]
```bash
~/work/lightbend/reactive-cli#master $ ldd cli/target/scala-2.11/rp 
	linux-vdso.so.1 (0x00007ffcae7aa000)
	librt.so.1 => /usr/lib/librt.so.1 (0x00007efe6f018000)
	libunwind.so.8 => /usr/lib/libunwind.so.8 (0x00007efe6edfe000)
	libunwind-x86_64.so.8 => /usr/lib/libunwind-x86_64.so.8 (0x00007efe6ebdf000)
	libhttpsimple.so => not found
	libre2.so.0 => /usr/lib/libre2.so.0 (0x00007efe6e973000)
	libcurl.so.4 => /usr/lib/libcurl.so.4 (0x00007efe6e6f1000)
	libstdc++.so.6 => /usr/lib/libstdc++.so.6 (0x00007efe6e36a000)
	libm.so.6 => /usr/lib/libm.so.6 (0x00007efe6e01e000)
	libgcc_s.so.1 => /usr/lib/libgcc_s.so.1 (0x00007efe6de07000)
	libc.so.6 => /usr/lib/libc.so.6 (0x00007efe6da50000)
	libpthread.so.0 => /usr/lib/libpthread.so.0 (0x00007efe6d832000)
	/lib64/ld-linux-x86-64.so.2 => /usr/lib64/ld-linux-x86-64.so.2 (0x00007efe6fb87000)
	liblzma.so.5 => /usr/lib/liblzma.so.5 (0x00007efe6d60c000)
	libnghttp2.so.14 => /usr/lib/libnghttp2.so.14 (0x00007efe6d3e6000)
	libssh2.so.1 => /usr/lib/libssh2.so.1 (0x00007efe6d1b8000)
	libpsl.so.5 => /usr/lib/libpsl.so.5 (0x00007efe6cfaa000)
	libssl.so.1.1 => /usr/lib/libssl.so.1.1 (0x00007efe6cd40000)
	libcrypto.so.1.1 => /usr/lib/libcrypto.so.1.1 (0x00007efe6c8c0000)
	libgssapi_krb5.so.2 => /usr/lib/libgssapi_krb5.so.2 (0x00007efe6c673000)
	libkrb5.so.3 => /usr/lib/libkrb5.so.3 (0x00007efe6c38d000)
	libk5crypto.so.3 => /usr/lib/libk5crypto.so.3 (0x00007efe6c15a000)
	libcom_err.so.2 => /usr/lib/libcom_err.so.2 (0x00007efe6bf56000)
	libz.so.1 => /usr/lib/libz.so.1 (0x00007efe6bd3f000)
	libicuuc.so.59 => /usr/lib/libicuuc.so.59 (0x00007efe6b98f000)
	libdl.so.2 => /usr/lib/libdl.so.2 (0x00007efe6b78b000)
	libkrb5support.so.0 => /usr/lib/libkrb5support.so.0 (0x00007efe6b57e000)
	libkeyutils.so.1 => /usr/lib/libkeyutils.so.1 (0x00007efe6b37a000)
	libresolv.so.2 => /usr/lib/libresolv.so.2 (0x00007efe6b163000)
	libicudata.so.59 => /usr/lib/libicudata.so.59 (0x00007efe69650000)
-> 0

```

**Comment by longshorej** [comment]
Closed via #17
